File: Examples/Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/Bookmarks.java
package com.aspose.words.model;

import java.util.ArrayList;
import java.util.List;
public class Bookmarks {
  private List<Bookmark> BookmarkList = new ArrayList<Bookmark>();
  private Link link = null;
  /**
	 * getBookmarkList
	 * Gets List<Bookmark>
	 * @return BookmarkList
	 */
  public List<Bookmark> getBookmarkList() {
    return BookmarkList;
  }

	/**
	 * setBookmarkList
	 * Sets List<Bookmark>
	 * @param BookmarkList List<Bookmark>
	 */
  public void setBookmarkList(List<Bookmark> BookmarkList) {
    this.BookmarkList = BookmarkList;
  }

  /**
	 * getLink
	 * Gets Link
	 * @return link
	 */
  public Link getLink() {
    return link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param link Link
	 */
  public void setLink(Link link) {
    this.link = link;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Bookmarks {\n");
    sb.append("  BookmarkList: ").append(BookmarkList).append("\n");
    sb.append("  link: ").append(link).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/src/Com/Aspose/Words/Model/Bookmarks.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Com.Aspose.Words.Model 
{
  /// <summary>
  /// 
  /// </summary>  
  public class Bookmarks 
  {           
        /// <summary>
        /// Initializes a new instance of the <see cref="Bookmarks" /> class.
        /// </summary>
        /// <param name="BookmarkList">BookmarkList.</param>
        /// <param name="Link">Link.</param>
        public Bookmarks(List<Bookmark> BookmarkList = default(List<Bookmark>), WordsApiLink Link = default(WordsApiLink))
        {
                        this.BookmarkList = BookmarkList;
                        this.Link = Link;
        }

        /// <summary>
        /// Gets or sets BookmarkList
        /// </summary>    
        public List<Bookmark> BookmarkList { get; set; }

        /// <summary>
        /// Gets or sets Link
        /// </summary>    
        public WordsApiLink Link { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class Bookmarks {\n");
          sb.Append("  BookmarkList: ").Append(BookmarkList).Append("\n");
          sb.Append("  Link: ").Append(Link).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
